.docker-compose.vital-strategies-theme.yaml

```diff
@@ -52,3 +56,7 @@ services:
                                    CREATE EXTENSION IF NOT EXISTS postgis_topology;         \
                                    CREATE EXTENSION IF NOT EXISTS fuzzystrmatch;            \
                                    CREATE EXTENSION IF NOT EXISTS postgis_tiger_geocoder;"
+  datapusher:
+    container_name: datapusher
+    build:
```

Comment:
This was already defined in the main `docker-compose.yaml`. Remove it from here, as it's redundant.